types/node/crypto.d.ts

```diff
@@ -429,7 +429,7 @@ declare module "crypto" {
     function createECDH(curve_name: string): ECDH;
     function timingSafeEqual(a: NodeJS.ArrayBufferView, b: NodeJS.ArrayBufferView): boolean;
     /** @deprecated since v10.0.0 */
-    const DEFAULT_ENCODING: string;
+    const DEFAULT_encoding: BufferEncoding;
```

Comment:
Find + replace issue?

Reply:
Good catch, I used a bad regex, I thought I caught all of these bad boys, ty :)